### DISCUSSION TITLE
How power up the ESP8266?

### DISCUSSION THREAD

**Discussion by DariBer** [awawa-dev/HyperSPI#5] [Q&A]
How does the ESP8266 get power(5V)? Is the USB cable connected between the unit running HyperHDR (in my case RPi4) and the ESP8266 card?

**Comment by awawa-dev** [comment]
Yes, it's best to connect ESP32/ESP8266 with Rpi using USB cable. Then you can also have an access from Rpi to HyperSPI frame rate statistics using serial connection.  That connection should provide also common ground between these two devices, but connecting theirs pinout grounds is still highly recommended.
